src/url.js

```diff
@@ -179,8 +180,15 @@ export function serializeQueryString(params) {
     const v = params[k];
     if (v == null) {
       continue;
+    } else if (isArray(v)) {
+      for (let i = 0; i < v.length; i++) {
+        const sv = /** @type {string} */ (v[i]);
+        s.push(`${encodeURIComponent(k)}=${encodeURIComponent(sv)}`);
```

Comment:
The standard form get is without `[]`, though people often put `[]` in the name of the input to indicate that more than one is expected.